blockFromPersonalCalendar.gs

```diff
@@ -55,14 +69,19 @@ function blockFromPersonalCalendar() {
       .map((event) => event.getId())
     );
   Object.values(knownEvents)
-    .filter((event) => !idsOnSecondaryCalendar.has(event.getTag(COPIED_EVENT_TAG)))
+    .filter((event) => !idsOnSecondaryCalendar.has(event.getTag(eventTag(calendarId))))
     .forEach((event) => {
       console.log(`Need to delete event on ${event.getStartTime()}, as it was removed from personal calendar`);
       event.deleteEvent();
   });
 }
 
-const COPIED_EVENT_TAG = 'blockFromPersonal.originalEventId';
+// const COPIED_EVENT_TAG = 'blockFromPersonal.originalEventId';
+
+const eventTag = (calendarId) => {
+  const calendarIdShort = calendarId.substring(0,5);
```

Comment:
any reason to make this shorten the id? (is there a limit on how long a tag can be?). It seems like this would ~easily get into conflicts (e.g. if I had marianosimone@ and marianossimone@)

Reply:
Great, thanks for confirming!